spec/models/note_spec.rb

```diff
@@ -14,4 +14,116 @@
   it 'has a valid factory' do
     expect(subject).to be_valid
   end
+
+  describe '.content_length' do
+    let(:north_utility) { create(:north_utility, code: 1) }
+    let(:south_utility) { create(:south_utility, code: 2) }
+
+    let(:north_user) { create(:user, utility_id: north_utility.id) }
+    let(:south_user) { create(:user, utility_id: south_utility.id) }
+
+    context 'when North Utility' do
+      let(:north_note) { create(:note, user_id: north_user.id, content: content, note_type: :critique) }
+
+      context 'with content <= 50 words' do
+        let(:content) { Faker::Lorem.words(number: 50).join(' ') }
+
+        it 'return "short"' do
+          expect(north_note.content_length).to eq('short')
+        end
+      end
+
+      context 'with content > 50 and content <= 100' do
+        let(:content) { Faker::Lorem.words(number: 100).join(' ') }
+
+        it 'return "medium"' do
+          expect(north_note.content_length).to eq('medium')
+        end
+      end
+
+      context 'with content > 100' do
+        let(:content) { Faker::Lorem.words(number: 101).join(' ') }
+
+        it 'return "long"' do
+          expect(north_note.content_length).to eq('long')
+        end
+      end
+    end
+
+    context 'when South Utility' do
+      let(:south_note) { create(:note, user_id: south_user.id, content: content, note_type: :critique) }
+
+      context 'with content <= 60 words' do
+        let(:content) { Faker::Lorem.words(number: 60).join(' ') }
+
+        it 'return "short"' do
+          expect(south_note.content_length).to eq('short')
+        end
+      end
+
+      context 'with content > 60 and content <= 120' do
+        let(:content) { Faker::Lorem.words(number: 120).join(' ') }
+
+        it 'return "medium"' do
+          expect(south_note.content_length).to eq('medium')
+        end
+      end
+
+      context 'with content > 120' do
+        let(:content) { Faker::Lorem.words(number: 121).join(' ') }
+
+        it 'return "long"' do
+          expect(south_note.content_length).to eq('long')
+        end
+      end
+    end
+  end
+
+  describe '.validate_content_length' do
+    let(:north_utility) { create(:north_utility, code: 1) }
+    let(:south_utility) { create(:south_utility, code: 2) }
+
+    let(:north_user) { create(:user, utility_id: north_utility.id) }
+    let(:south_user) { create(:user, utility_id: south_utility.id) }
```

Comment:
Mismo comentario que antes, acá lo que queremos es tener una nota de contenido variable, no?

```suggestion
    let(:user) { create(:user, utility: utility) }
    let(:note) { build(:note, user: user, content: content, note_type: :review) }
```

Además fjiate que para las factories no hace falta que pasemos y asignemos el `id`, podemos pasar directamente el `user` . Son las ventajas que nos da el framework que internamente sabe como guardar esa relación entre note y user en la base de datos 🚀

Reply:
Hola @florfrioli, gracias por todas las recomendaciones. Ya quedaron los cambios realizados.